_tutorials/page2.md

```diff
@@ -1,17 +1,74 @@
 ---
 layout: tutorial
-title: Tutorial
-name: Creating custom templates
-previous_page_title: How to install
-next_page_title: Chaining requests
+title: API Sign Up
+name: API Sign Up
+previous_page_title: Setup
+next_page_title: Run
 ---
 
-## Part 2!
+# Snippets API Sign Up
 
-```python
-x + 2 = 3
+This tutorial will use our demo API, Snippets API, to show how to use ScanAPI. Snippets API was
+built based on the [Django REST Framework's tutorial][drf-tutorial]. You can access it at
+[demo.scanapi.dev/](https://demo.scanapi.dev/).
+
+The main goal of this API is to manage code snippets. Each code snippet have the corresponding
+syntax highlight for it, based on the language, linenos and style.
+
+This is an example of a code snippet:
+[demo.scanapi.dev/api/v1/snippets/2/](https://demo.scanapi.dev/api/v1/snippets/2/)
+
+```json
+{
+  "url": "http://demo.scanapi.dev/api/v1/snippets/2/",
+  "id": 2,
+  "highlight": "http://demo.scanapi.dev/api/v1/snippets/2/highlight/",
+  "owner": "admin",
+  "title": "Calculator",
+  "code": "def add(x, y):\r\n    return x + y\r\n\r\ndef subtract(x, y):\r\n    return x - y\r\n\r\ndef multiply(x, y):\r\n    return x * y\r\n\r\ndef divide(x, y):\r\n    return x / y",
+  "linenos": true,
+  "language": "python",
+  "style": "vs"
+}
 ```
 
-Duis congue porttitor blandit. Fusce euismod orci in enim rhoncus, vitae porta sapien sagittis. Integer vitae est sit amet sapien vulputate accumsan ac eu dui. Suspendisse quis elit vitae quam tincidunt tincidunt vel nec tellus. Integer ornare erat eget libero mattis porta. Donec mollis tortor ante, sit amet pretium risus pharetra faucibus. Maecenas tellus enim, commodo at ligula nec, sollicitudin auctor eros. Donec consectetur leo urna, vitae sollicitudin dolor ullamcorper sodales. Aliquam id nisi lorem. Fusce tincidunt congue libero, in cursus erat auctor sed. Suspendisse potenti. Curabitur efficitur, est et sollicitudin pellentesque, leo dolor feugiat turpis, et faucibus nunc quam vitae justo. Etiam id nibh mauris. Donec molestie auctor dapibus. Quisque vehicula laoreet justo et tristique.
+and this is its highlighted version: [demo.scanapi.dev/api/v1/snippets/2/highlight/](https://demo.scanapi.dev/api/v1/snippets/2/highlight/)
+
+<p>
+  <img
+    src="/assets/images/tutorial/page2/calculator-highlight.png"
```

Comment:
@camilamaia we can use something like https://carbon.now.sh/ for generating this image. Would look a bit cleaner and prettier.

I tried the same code with a light theme that would match with the website. But feel free to try any other theme if you like, 

![image](https://user-images.githubusercontent.com/13925830/120751612-f46a4880-c525-11eb-8432-5158440fa5c3.png)

Reply:
@Pradhvan 100% agreed, the image sucks now 😂  But, the point here is to use the demo API to get the syntax highlight. If we use carbon, or something similar, we would not need the API.

With that in mind, we definitely can do a better a job here hahah I changed the `Style` and improved the screenshot, let me know what you think.

Also, if you did not like the option I proposed, there is also these ones that I think could be a good fit:

**Arduino**

![image](https://user-images.githubusercontent.com/2728804/120814022-d6501880-c524-11eb-932e-9461a0dd2768.png)

**Emacs** (the one I am currently using)

![image](https://user-images.githubusercontent.com/2728804/120814084-e5cf6180-c524-11eb-9c17-195d58357c95.png)

**Manni**

![image](https://user-images.githubusercontent.com/2728804/120814130-f384e700-c524-11eb-83e7-770e64f13d60.png)

**Xcode**

![image](https://user-images.githubusercontent.com/2728804/120814159-fb448b80-c524-11eb-9320-cd0d448e9e29.png)

Let me know your thoughts!

Reply:
> @Pradhvan 100% agreed, the image sucks now joy But, the point here is to use the demo API to get the syntax highlight. If we use carbon, or something similar, we would not need the API.

I agree but since it's only a demo API so use carbon or not shouldn't matter much. My point was that we can style up the image.